python-checks/src/test/resources/checks/changeMethodContract.py

```diff
@@ -118,3 +118,9 @@ def read(self): ... # Noncompliant {{Add 1 missing parameter. This method overri
 
 class MyTextIOWrapper(TextIOWrapper):
   def seek(self): ... # Noncompliant {{Add 2 missing parameters. This method overrides io.TextIOWrapper.seek.}}
+
+import datetime
+
+class MyTZInfo(datetime.tzinfo):
+  def tzname(self): # Noncompliant
+    ...
```

Comment:
I was thinking, shouldn't this raise as well a ArgumentNumber issue? Or is the test on line 71 of argumentNumber.py at the wrong place?